lib/Cargo.toml

```diff
@@ -106,8 +106,8 @@ sha2 = "0.10.6"
 speedb = { version = "0.0.2", optional = true }
 storekey = "0.5.0"
 thiserror = "1.0.40"
-tikv = { version = "0.1.0", package = "tikv-client", optional = true }
-tikv-client-proto = { version = "0.1.0", optional = true }
+tikv = { version = "0.1.0-surreal.1", package = "surrealdb-tikv-client", optional = true }
+tikv-client-proto = { version = "0.1.0-surreal.1", package = "surrealdb-tikv-client-proto", optional = true }
```

Comment:
I've updated our fork of TiKV rust client to have some additional functionalities needed by Change Feed:

https://github.com/surrealdb/tikv-client/tree/0.1.0-surreal.x

and published all the related crates to crates.io so that we can depend on the crates this way

https://crates.io/search?q=surrealdb-tikv

I didn't use `git` dependency in Cargo.toml as that would prevent us from publishing surrealdb as a crate.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories